libs/openFrameworksCompiled/project/osx/CoreOF.xcconfig

```diff
@@ -1,3 +1,19 @@
+ARCHS = $(ARCHS_STANDARD_32_64_BIT)	// this allows you to select target architecture in dropdown menu
+VALID_ARCHS = x86_64 i386
+
+ONLY_ACTIVE_ARCH = YES      // this makes sure not to build an universal binary
```

Comment:
It does (and in most cases that's what you probably want). You can still build a fat (32+64bit) app binary if you must, by unchecking the "build only active architecture" in the project and openFrameworksLib target build settings (these override the config files). 

I've set up the templates in a way to make it easier for beginners to switch. But advanced users can still override the defaults by setting the target build settings on a per-project basis.

That said, 32/64 fat app code can only be written in cpp98 (like openFrameworks core) so that the i386 slice will still compile (the 64 bit slice will be compiled as c++11, which is backwards compatible with cpp98).